src/data/senoppfolging/useSenoppfolgingSvarQuery.ts

```diff
@@ -0,0 +1,22 @@
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { MEROPPFOLGING_BACKEND_ROOT } from "@/apiConstants";
+import { get } from "@/api/axios";
+import { useQuery } from "@tanstack/react-query";
+import { minutesToMillis } from "@/utils/timeUtils";
+
+export const senoppfolgingSvarQueryKeys = {
```

Comment:
`senoppfolging` eller `senOppfolging` 🤔🧠💥
```suggestion
export const senOppfolgingSvarQueryKeys = {
```

Reply:
Blir vel senOppfolging ja